subsys/bluetooth/host/adv.c

```diff
@@ -1469,6 +1488,17 @@ void bt_le_adv_resume(void)
 	if (IS_ENABLED(CONFIG_BT_PRIVACY) &&
 	    !atomic_test_bit(adv->flags, BT_ADV_USE_IDENTITY)) {
 		bt_id_set_adv_private_addr(adv);
+	} else {
+		uint8_t own_addr_type;
+		bool dir_adv = !bt_addr_le_eq(&adv->target_addr, BT_ADDR_LE_ANY);
```

Comment:
Perhaps outside the scope of this PR: I think it would be nice to have this in a function or macro; it seems to just be implied that we use `BT_ADDR_LE_ANY` for nondirected advertising.

Reply:
Yeah that's the easiest way I thought of sensing this. The advertiser needs some refactoring, and we're going to do that this year. I'll add a macro though, it makes the code more readable.